test/JZSolverTests.cpp

```diff
@@ -0,0 +1,49 @@
+#include <gtest/gtest.h>
+#include "JZSolver.hpp"
+#include <memory>
+
+struct JZSolverTests : public ::testing::Test
+{
+    JZSolver solver;
+};
+
+TEST_F(JZSolverTests, checkJZSolverConstructor)
+{
+    Board board_col_err(4,
```

Comment:
You can format every Board like this:
```cpp
Board board_name(
    4,
    3, 
    {...},
    {...}};
```

It will be more consistent, because every board will have the same indents

Reply:
done